collapse_long_paths.go

```diff
@@ -24,7 +24,7 @@ func CollapseLongPathsFromNode(t *Tree, nodeName string) {
 	for children := t.To[q]; len(children) == 1; children = t.To[q] {
 		nextChild := children[0]
 
-		parts = append(parts, t.Nodes[q].Name)
+		parts = append(parts, slashToEntity.Replace(t.Nodes[q].Name))
```

Comment:
not sure we need to replace here. name is just one of parts. it should be already acceptable